pkgs/servers/nextcloud/packages/default.nix

```diff
@@ -6,11 +6,12 @@
   lib,
   pkgs,
   newScope,
-  apps,
```

Comment:
Hi! At a glance, this change seems to make it impossible to override the json manifests out-of-tree (I was using this in my overlays). If that's correct, here's a hotfix: https://github.com/NixOS/nixpkgs/pull/389371